GLib-2.0.gir

```diff
@@ -51744,7 +51754,7 @@ are different concepts on Windows.</doc>
           <doc xml:space="preserve">flags from #GSpawnFlags</doc>
           <type name="SpawnFlags" c:type="GSpawnFlags"/>
         </parameter>
-        <parameter name="child_setup" transfer-ownership="none" nullable="1" allow-none="1" scope="async" closure="5">
+        <parameter name="child_setup" transfer-ownership="none" scope="async" closure="5">
```

Comment:
Like written in the regenerate PR, this is wrong. The callback is optional according to the code.